README.md

```diff
@@ -1 +1,58 @@
-# hgj-getting-started
+Local Game Jam Starter Kit
+==========================
+
+This kit will help you get started.  We want all contributions to be free as in freedom and so you do not need to worry about licensing too much, you can make use of this repo.  We provide information for physical contributions as well as for software contributions.
+
+Physical Contributions
```

Comment:
Since software contributions are far more common, I'd put those first and the physical contributions second. What do you think?

Reply:
Agreed.  However, I put the physical one first because it is shorter and I didn't want it to get lost.  Do you think it would be better otherwise?

Reply:
Ok, I see. Since the software part also isn't that long, I don't think that the physical part would get lost as the descriptions are pretty concise :) Also the headlines make it pretty clear where to look for the correct part I think. So personally I don't think that it would be a problem to put it the other way around.

Reply:
Done.